pkg/agent/route/route_windows.go

```diff
@@ -387,7 +387,7 @@ func (c *Client) DeleteSNATRule(mark uint32) error {
 
 // TODO: nodePortAddresses is not supported currently.
 func (c *Client) AddNodePort(nodePortAddresses []net.IP, port uint16, protocol binding.Protocol) error {
-	return util.ReplaceNetNatStaticMapping(antreaNatNodePort, "0.0.0.0", port, config.VirtualServiceIPv4.String(), port, string(protocol))
+	return util.ReplaceNetNatStaticMapping(antreaNatNodePort, "0.0.0.0", port, config.VirtualNodePortDNATIPv4.String(), port, string(protocol))
```

Comment:
should this comment be corrected?
https://github.com/antrea-io/antrea/blob/d1c6a43c4f5dcda0b1c2dc837310e076bb9094d2/pkg/agent/config/node_config.go#L54-L57

Reply:
https://github.com/antrea-io/antrea/blob/d1c6a43c4f5dcda0b1c2dc837310e076bb9094d2/pkg/agent/config/node_config.go#L61

Reply:
I mean this line "The IP is used for InternalIPAddress parameter of Add-NetNatStaticMapping on Windows." I think it means the usage you are changing here?

Reply:
Yes, it should be removed now, thanks.

Reply:
Updated.